Content.Shared/Chemistry/Reaction/SharedChemicalReactionSystem.cs

```diff
@@ -13,18 +13,60 @@ namespace Content.Shared.Chemistry.Reaction
 {
     public abstract class SharedChemicalReactionSystem : EntitySystem
     {
-        private IEnumerable<ReactionPrototype> _reactions = default!;
 
+        /// <summary>
+        /// 
+        /// </summary>
         private const int MaxReactionIterations = 20;
 
         [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
         [Dependency] private readonly IRobustRandom _random = default!;
-        [Dependency] protected readonly SharedAdminLogSystem _logSystem = default!;
+        [Dependency] protected readonly SharedAdminLogSystem LogSystem = default!;
```

Comment:
1) why can't this be private and 2) it should be _logSystem, VS is wrong

Reply:
1. The ChemicalReactionSystem subtype used by the server accesses it so it can't be private.
2. Alright. I'll look into how to change VS's highlighting.

Reply:
Currently the server-side `ChemicalReactionSystem` that inherits from this also uses the log system, so I guess that's why its not private. And I'm pretty sure PascalCase is still used for protected.